### PR TITLE
Audacious aardvark

### PR DESCRIPTION
**Pull request by catlook**
Edited multiple blogs to remove references to the Rackspace Marketplace or Cloud Tools Marketplace and Rackspace partners.

### CODE REVIEW

File: _posts/2013-06-19-using-new-relic-with-clb.markdown [hunk 1]
```diff
@@ -35,15 +35,15 @@ Next, run a few commands from within this director to get the gem set up:
...
 
 **NOTE**: It is crucial to use lowercase for the region – it will not work otherwise. If you have load balancers in multiple regions, you will need to configure a separate plugin per region.
 
-Start the agent: `./bin/newrelic_rs`. The correct output here should say “gathering xx statistics” 
+Start the agent: `./bin/newrelic_rs`. The correct output here should say “gathering xx statistics”
 
 Now you can log in to your New Relic account and you’ll see the Rackspace Cloud Load Balancer icon in the left navigation pane. Once there you will see your load balancer(s) and you can view traffic, configure alerts, and you are good to go!
 
-Click [here][2] to start using the plugin today and keep an eye out for more Rackspace plugins that are currently under development. If you don’t already have a New Relic account, all Rackspace customers get a free Standard account for life through our Cloud Tools Marketplace. 
+Click [here][2] to start using the plugin today and keep an eye out for more Rackspace plugins that are currently under development.
```
Comment (DonNateR) [inline _posts/2013-06-19-using-new-relic-with-clb.markdown]: Is there a way to verify that these steps still work with Relic? There's a possibility that we might have to retire the entire article.

File: _posts/2013-05-22-qbox.markdown [hunk 2]
```diff
@@ -143,7 +143,7 @@ puts search.results.map(&:title)
 
 With a bit of JavaScript knowledge and a library like [Twitter Typeahead][11], you’re only a few minutes away from a sexy auto-complete feature!
 
-Qbox has a free developer tier with one shard per index, and should be available soon on the [Rackspace Cloud Tools Marketplace][12]. It can be launched from the Marketplace, or directly from the qbox.io website. Finally, for readers of the DevOps blog, [StackSearch CEO Mark Brandon][13] has given us a promo code for some free 90-day trials on any of the plan tiers, including the Enterprise plan that comes with four shards per index and three replicas - just use the [promo code "RAX"][14] when checking out. 
+Qbox has a free developer tier with one shard per index. It can be launched directly from the qbox.io website. Finally, for readers of the DevOps blog, [StackSearch CEO Mark Brandon][12] has given us a promo code for some free 90-day trials on any of the plan tiers, including the Enterprise plan that comes with four shards per index and three replicas - just use the [promo code "RAX"][13] when checking out.
```
Comment (DonNateR) [inline _posts/2013-05-22-qbox.markdown]: Does the promo-code still apply? This could be another article we need to retire.

**Comment by catlook** [comment]
Thanks Nate ...  I'll publish as is (once i fix the strider build) and then circle back to delete the article!

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-fd5e5e00e0/)

**Comment by smashwilson** [comment]
More network gremlins it looks like.

¯\\_(ツ)_/¯

**Pull request merged**